src/vulkan/engine_vulkan.cc

```diff
@@ -450,6 +450,85 @@ Result EngineVulkan::DoDrawRect(const DrawRectCommand* command) {
   return {};
 }
 
+Result EngineVulkan::DoDrawGrid(const DrawGridCommand* command) {
+  auto& info = pipeline_map_[command->GetPipeline()];
+  if (!info.vk_pipeline->IsGraphics())
+    return Result("Vulkan::DrawGrid for Non-Graphics Pipeline");
+
+  auto* graphics = info.vk_pipeline->AsGraphics();
+
+  float x = command->GetX();
+  float y = command->GetY();
+  float width = command->GetWidth();
+  float height = command->GetHeight();
+  const uint32_t columns = static_cast<uint32_t>(command->GetColumns());
+  const uint32_t rows = static_cast<uint32_t>(command->GetRows());
+  const uint32_t vertices = columns * rows * 3 * 2;
```

Comment:
Let's move the 3 and 2 to constants which will make this a lot more obvious

```
kTrianglesPerCell = 2
kVerticesPerTriangle = 3
```

Reply:
Done. Why are these constants prefixed with k, but not others?

Reply:
Mostly I was thinking they would move up to the anonymous namespace at the top of the file making them constants, which we prefix with a k. Leaving them in the method you could drop the k if you want, either way is fine.